apps/data-types.md

```diff
@@ -577,6 +577,12 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | - | - |  - |
 | quantity | number | The quantity of the product line item displayed as a decimal number.|
 
+## productLineItemPrice (update request)
+
+| Attribute | Type | Description |
+| - | - |  - |
+| singleItemPrice | number | The price for a single item.|
```

Comment:
```suggestion
| singleItemPrice | number | The price of a single product line item.|
```

Reply:
I'm not sure I fully understood the difference between `singleItemPrice` and `lineItemPrice`.
Example:  I have 3 pcs. of chocolate in the cart. Price for those 3 pcs. is 9€ = lineItemPrice? And 1 pc. of chocolate is 3€ = singleItemPrice?
@manel-ePages can you shed some light on this?

Reply:
Yes, it's exactly as you describe. One example:
`"productLineItems": [
            {
                "lineItemId": "5DB0676A-F14A-8E54-D484-D5809A8D9C08",
                "sku": "01-handbag",
                "name": "Handbag - brown",
                "productId": "5DA83046-48EA-3C13-1507-D5809A8D293B",
                "quantity": {
                    "amount": 2,
                    "unit": "piece(s)"
                },
                "lineItemPrice": {
                    "taxType": "GROSS",
                    "formatted": "339.90 €",
                    "amount": 339.9,
                    "currency": "EUR"
                },
                "singleItemPrice": {
                    "taxType": "GROSS",
                    "formatted": "169.95 €",
                    "amount": 169.95,
                    "currency": "EUR"
                },`